hypothesis-python/tests/cover/test_datetimes.py

```diff
@@ -111,9 +111,9 @@ def test_can_find_before_the_year_2000():
     assert minimal(dates(), lambda x: x.year < 2000).year == 1999
 
 
-def test_can_find_each_month():
-    for month in hrange(1, 13):
-        find_any(dates(), lambda x: x.month == month)
+@pytest.mark.parametrize("month", hrange(1, 13))
+def test_can_find_each_month(month):
+    find_any(dates(), lambda x: x.month == month, settings(max_examples=10 ** 6))
```

Comment:
I’m surprised we need as many examples for this.

Back of a napkin: probability of not choosing a given month in N samples is ~(11/12)^N, I think? So 200 samples already gets the probably down to 2.8e-08.

Reply:
Also surprised, but there we are... I think it might be better to refactor the implementation of `dates()` to draw year/month/day (like `datetimes()`) rather than a number-of-days offset from the center.